lib/cli/clients/ega/ega_script.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/python
+import subprocess
+import random
+import string
+import logging
+
+
+def ega_call(id, passfile, toolpath, udt, downloaddirectory):
+    logger = logging.getLogger('__log__')
+
+    lable = id + '_request'
+    key = ''.join(random.SystemRandom().choice(string.ascii_uppercase + string.digits) for _ in range(4))
+    if id[4] == 'd':
+        downloadtype = '-rfd'
+    else:
+        downloadtype = '-rf'
+    requestcall_args = ['java', '-jar', toolpath, '-pf', passfile, downloadtype, id, '-re', key,
```

Comment:
I would name this `request_call_args`

A good reference for python conventions: https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables